Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -3178,6 +3179,19 @@ - (void)_addAppHighlightInNewGroup:(BOOL)newGroup originatedInApp:(BOOL)originat
 #endif
 }
 
+- (void)_requestAllTextWithCompletionHandler:(void(^)(NSArray<_WKTextRun *> *))completionHandler
+{
+    _page->requestAllTextAndRects([weakSelf = WeakObjCPtr<WKWebView>(self), completion = makeBlockPtr(completionHandler)](auto&& textRuns) {
+        auto strongSelf = weakSelf.get();
+        if (!strongSelf)
+            return completion(@[ ]);
+
+        completion(createNSArray(textRuns, [](auto& textRun) {
```

Comment:
I think you can `std::forward(textRuns)` so if you're passing in an rvalue reference it can get moved along.

Reply:
Hm... that gave me a compile-time error:

```
OpenSource/Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm:3301:34: error: no matching function for call to 'forward'
 3301 |         completion(createNSArray(std::forward(textRuns), [](auto& textRun) mutable {
```

I'll adjust this to instead `WTFMove(textRuns)` when passing into `createNSArray`, to take advantage of move semantics here.